src/common/libflux/msg_handler.h

```diff
@@ -27,7 +27,7 @@ void flux_msg_handler_deny_rolemask (flux_msg_handler_t *w, uint32_t rolemask);
 
 struct flux_msg_handler_spec {
     int typemask;
-    char *topic_glob;
+    const char *topic_glob;
```

Comment:
Maybe this was pulled in by mistake?  If intended, it should probably be split out to another commit.

Reply:
Actually, it was intentional. It was required to allow the header to compiler under the C++ compiler.  I   should note that in the comment.